Python/import.c

```diff
@@ -387,11 +387,32 @@ import_get_module(PyThreadState *tstate, PyObject *name)
 }
 
 
-PyObject *
-PyImport_GetModule(PyObject *name)
+static int
+import_ensure_initialized(PyThreadState *tstate, PyObject *mod, PyObject *name)
```

Comment:
FWIW, I always prefer to pass in as little as possible.  Here `PyThreadState` is quite broad, since we only need `PyInterpreterState.importlib`.  At the same time, for smaller code like this it can help to keep the common indirections in the function.  So I guess I'd leave it as-is. :)